website/src/components/Header/Header.tsx

```diff
@@ -25,7 +25,7 @@ export const Header = () => {
 
   return (
     <nav className="fixed w-full z-20 top-0 start-0 bg-[#F7F8F8]/1 border-b border-[#F7F8F8]/8 backdrop-blur-[20px]">
-      <div className="flex flex-wrap items-center justify-between mx-auto px-6 py-4">
+      <div className="flex flex-wrap items-center justify-between mx-auto px-6 md:px-20 py-4">
```

Comment:
can we slightly reduce the vertical padding for the header and also the sign-up button? overall looks good but the header seems slightly too tall

```suggestion
      <div className="flex flex-wrap items-center justify-between mx-auto px-6 md:px-20 py-2">
```